linking/runtime-linking/resolve-function-by-fin8-fasthash.yml

```diff
@@ -0,0 +1,38 @@
+rule:
+  meta:
+    name: resolve function by FIN8 fasthash
+    namespace: linking/runtime-linking
+    author: "@r3c0nst (Frank Boldewin)"
+    description: APIHashing algorithm derived from a fasthash implementation in OpenCPN using seeds
+    scope: function
+    references:
+      - https://www.bitdefender.com/files/News/CaseStudies/study/394/Bitdefender-PR-Whitepaper-BADHATCH-creat5237-en-EN.pdf
+      - https://raw.githubusercontent.com/fboldewin/YARA-rules/master/Shellcode.APIHashing.FIN8.yar
+    examples:
+      - B43FCA5283BFC7022553EFF663683834:0x140
+      - 4BF70EA92979DD88C9761EE848370050:0x28b
+  features:
+    - or:
+      - basic block:
+        - and:
+            # 64-bit constants
+          - mnemonic: mov
+          - number: 0x880355F21E6D1965
+          - number: 0x2127599BF4325C37
+          - mnemonic: add
+      - or:
```

Comment:
i think this `or` is redundant. but not a big deal.

Reply:
removed this